src/Core/Models/CustomCommands/CustomCommand.cs

```diff
@@ -11,6 +11,7 @@ public class CustomCommand
     {
         public string Name { get; set; }
         public List<CustomCommandFeature> Features { get; set; }
+        public bool IgnoreArguments { get; set; }
```

Comment:
Just to make sure, for now this is always `true`, and we can later on replace it with a `ParameterFeature` or whatever, right?

Reply:
Yes, during command creation this is default set to true, during fetching I return this value or true.